app/src/main/java/de/tum/in/tumcampusapp/utils/Utils.kt

```diff
@@ -72,6 +72,22 @@ object Utils {
         return sp.getString(key, defaultVal)!!
     }
 
+    /**
+     * Get a value from the default shared preferences.
+     *
+     * TODO: This method is unused. Should we keep it?
```

Comment:
This will have to be decided in the review

Reply:
No.